lib/management/push.js

```diff
@@ -244,7 +254,8 @@ function startWSManagementClient(pushEndpoint, instanceId, token) {
             closeChannel(message.getChannelNumber());
             break;
         case CHANNEL_PAYLOAD_MESSAGE:
-            if (_config.browserAccessEnabled) {
+            // browserAccessEnabled defaults to true unless explicitly false
+            if (_config.browserAccessEnabled === undefined || _config.browserAccessEnabled) {
```

Comment:
minor style: maybe you could write `if (_config.browserAccessEnabled !== false)`

Reply:
of course 🤦‍♂